docs/scripts_md/dicomTar.md

```diff
@@ -0,0 +1,70 @@
+# NAME
+
+dicomTar.pl -- archives DICOM data into the LORIS database
+
+# SYNOPSIS
+
+perl dicomTar.pl &lt;/PATH/TO/SOURCE/DICOM> &lt;/PATH/TO/TARGET/DIR> \`\[options\]\`
+
+Available options are:
+
+\-today                  : Use today's date for archive name instead of using
+                          acquisition date
+
+\-database               : Use a database if you have one set up for you.
+                          Just trying will fail miserably
+
+\-mri\_upload\_update      : update the mri\_upload table by inserting the correct
```

Comment:
To be consistent with other descriptions, "update" should be written "Update the mri_upload table by inserting...)